simplq/src/components/pages/Admin/ShareQueue.jsx

```diff
@@ -26,4 +46,87 @@ const ShareQueue = (props) => {
   );
 };
 
+const ShareQueue = (props) => {
+  const [open, setOpen] = useState(false);
+  const anchorRef = useRef(null);
+
+  const handleToggle = () => {
+    setOpen((prevOpen) => !prevOpen);
+  };
+
+  const handleClose = (event) => {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
+      return;
+    }
+    setOpen(false);
+  };
+
+  const quote = `Hi! Use ${window.location.origin}/j/${props.queueName} to join my queue and get live updates.`;
+  const link = `${window.location.origin}/j/${props.queueName}`;
+
+  return (
+    <>
+      <ButtonGroup
+        variant="contained"
+        className={styles['button-group']}
+        ref={anchorRef}
+        aria-label="split button"
+      >
+        <CopyButton {...props} />
+        <Button
+          className={styles['button-group']}
+          color="primary"
+          size="small"
+          onClick={handleToggle}
+        >
+          <ArrowDropDownIcon />
+        </Button>
+      </ButtonGroup>
+      <Popper
+        style={{ zIndex: 100, backgroundColor: '#fff' }}
+        open={open}
+        anchorEl={anchorRef.current}
+        transition
+        disablePortal
+        placement="bottom-end"
+      >
+        {({ TransitionProps, placement }) => (
+          <Grow
+            {...TransitionProps}
+            style={{
+              transformOrigin: placement === 'bottom' ? 'center top' : 'center bottom',
+            }}
+          >
+            <Paper>
+              <ClickAwayListener onClickAway={handleClose}>
+                <MenuList>
+                  Share
```

Comment:
Let's remove this text, it's pretty obvious even without it. What do you think?

Reply:
We can keep it